deployment/scripts/start-bootstrap.sh

```diff
@@ -2,8 +2,24 @@
 
 cd lotus || exit
 
+# Kill a potentially running instance of Lotus
 tmux kill-session -t lotus
 tmux new-session -d -s lotus
-tmux send-keys "./lotus daemon --genesis=spacenet-genesis.car --profile=bootstrapper 2>&1" C-m
+
+# Start the Lotus daemon and import the bootstrap key.
+# Keeping the version with a custom genesis commented, in case we need to come back to it.
+#tmux send-keys "./lotus daemon --genesis=spacenet-genesis.car --profile=bootstrapper --bootstrap=false 2>&1" C-m
+tmux send-keys "./lotus daemon --profile=bootstrapper --bootstrap=false 2>&1" C-m
+mkdir -p ~/.lotus/keystore && chmod 0700 ~/.lotus/keystore
+./lotus-shed keyinfo import spacenet-libp2p-bootstrap1.keyinfo
+echo '[Libp2p]
+ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]' > ~/.lotus/config.toml
 ./lotus wait-api
 ./lotus net listen | grep -vE '(/ip6/)|(127.0.0.1)' | grep -E '/ip4/.*/tcp/' > ~/.lotus/lotus-addr
+
+# Start the Faucet.
+./lotus wallet import --as-default --format=json-lotus spacenet_faucet.key
+cd ~/spacenet/faucet/ || exit
+go build -o spacenet-faucet ./cmd || exit
```

Comment:
There's a PR pending from @dnkolegov (https://github.com/consensus-shipyard/spacenet/pull/10) where he changed the defaults so we may have to add flags here to point to our key and the default port of our Lotus node, but we can merge this and fix it in the next one.

Reply:
Agree. But I think it would be best to merge this, rebase Denis' PR on top of it, and add a commit directly to that PR to use the appropriate flags here.